python/py-urwid_readline/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        rr- urwid_readline 0.15.1
+name                py-${name}
+revision            0
+
+license             MIT
```

Comment:
it looks like this is a pure Python package and nothing gets compiled and/or is architecture-dependent. If that's indeed the case you should add:
```
plaforms        {darwin any}
supported_archs noarch
```